### PR TITLE
Changes to make it work for OS X

### PR DESCRIPTION
**Pull request by snukavarapu**
These are the changes made to make it work on the OS X. 

https://github.com/bloomberg/chef-bach/issues/663

### CODE REVIEW

File: tests/automated_install.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ done
 
 echo "Checking for GNU screen to watch serial consoles"
 if hash screen 2>/dev/null ; then
-  pushd $(readlink -f $(dirname $0))
+  pushd $(greadlink -f $(dirname $0))
```
Comment (cbaenziger) [inline tests/automated_install.sh]: This looks good; but I would suggest that there be a conditional for the `readlink` binary. Perhaps setting a variable for the `readlink` binary's name? This way the current binary name can be used on Linux and `greadlink` on OS X?

**Comment by cbaenziger** [comment]
This is a good start! Perhaps you could update [README.md](https://github.com/bloomberg/chef-bach/blob/master/README.md) to explain what OS X users should install; e.g. `brew` and `greadlink`.

**Comment by snukavarapu** [comment]
@cbaenziger 

https://github.com/bloomberg/chef-bach/compare/master...snukavarapu:master

Please check tests/automated_install.sh @ https://github.com/bloomberg/chef-bach/compare/master...snukavarapu:master

I have added 'brew install coreutils' for Mac, it tries to install Coreutils for "greadlink" to work. If it is already there it throws warning like "Warning: coreutils-8.25 already installed".

I have tested it and did run till the end "Full-shoes".

**Comment by snukavarapu** [comment]
https://github.com/bloomberg/chef-bach/compare/master...snukavarapu:master#diff-2315a606c13a41b94b167512d8191fb3


I have changed tests/automated_install.sh so it will try to install 'brew install coreutils' for mac users. If it is already installed it throws warning like "Warning: coreutils-8.25 already installed".

I have tested and it worked and the cluster build till "full-shoes"

**Comment by snukavarapu** [comment]
https://github.com/bloomberg/chef-bach/compare/master...snukavarapu:master#diff-2315a606c13a41b94b167512d8191fb3


I have changed tests/automated_install.sh so it will try to install 'brew install coreutils' for mac users. If it is already installed it throws warning like "Warning: coreutils-8.25 already installed".

I have tested and it worked and the cluster build till "full-shoes"

File: tests/automated_install.sh [hunk 2]
```diff
@@ -64,7 +64,12 @@ done
 
 echo "Checking for GNU screen to watch serial consoles"
 if hash screen 2>/dev/null ; then
-  pushd $(readlink -f $(dirname $0))
+  if [ "$(uname)" == "Darwin" ]; then
+   brew install coreutils
```
Comment (cbaenziger) [inline tests/automated_install.sh]: Please use two spaces for indents

**Comment by cbaenziger** [comment]
This can't be merged (see the conflict message here on the pull-request) as ` setup_chef_cookbooks.sh ` was corrected in Chef-BACH #674. Could you remove the changes to this file from your commit?

**Comment by snukavarapu** [comment]
I have removed my changes to setup_chef_cookbooks.sh. 

Here is new pull-request:
https://github.com/bloomberg/chef-bach/compare/master...snukavarapu:master

**Comment by cbaenziger** [comment]
Hi Sai, could you please update the readme to suggest OS X folks have `brew` installed, unless it's now standard on OS X?

**Comment by snukavarapu** [comment]
Hi Clay, Sure! It looks like brew is not standard on OS X, so I have added to readme file.

**Pull request closed without merging**